app/src/main/res/values/styles.xml

```diff
@@ -1,5 +1,5 @@
 <!--
```

Comment:
Is this file still required?

Reply:
it is per the discussion in the best practices doc - but it should be renamed to themes.xml